python/semantic_kernel/utils/model_diagnostics.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) Microsoft. All rights reserved.
+#
+# Model diagnostics to trace model activities with the OTel semantic conventions.
+# This code contains experimental features and may change in the future.
+# To enable these features, set one of the following senvironment variables to true:
+#    SEMANTICKERNEL_EXPERIMENTAL_GENAI_ENABLE_OTEL_DIAGNOSTICS
+#    SEMANTICKERNEL_EXPERIMENTAL_GENAI_ENABLE_OTEL_DIAGNOSTICS_SENSITIVE
+
+import json
+import os
+from typing import Optional
+
+from azure.monitor.opentelemetry import configure_azure_monitor
+from opentelemetry import trace
+from opentelemetry.trace import Span
+
+from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
+from semantic_kernel.contents import ChatHistory
+from semantic_kernel.contents.chat_message_content import ITEM_TYPES, ChatMessageContent
+from semantic_kernel.contents.function_call_content import FunctionCallContent
+
+# Activity tags
+_SYSTEM = "gen_ai.system"
```

Comment:
this should be in a const file, maybe utils/tracing/const.py or even utils/opentelemetry/const